src/treemodel/treewalker.js

```diff
@@ -214,19 +341,32 @@ function formatReturnValue( type, item, previousPosition, nextPosition, length )
  */
 
 /**
- * Object returned by {@link engine.treeModel.TreeWalker} when traversing tree model.
+ * Object returned by {@link engine.treeModel.TreeWalker} when traversing model.
  *
  * @typedef {Object} engine.treeModel.TreeWalkerValue
  * @property {engine.treeModel.TreeWalkerValueType} type
  * @property {engine.treeModel.Item} item Item between old and new positions of {@link engine.treeModel.TreeWalker}.
- * @property {engine.treeModel.Position} previousPosition Previous position of the iterator. For `'ELEMENT_END'` it is the last
- * position inside the element. For all other types it is the position before the item. Note that it is more
- * efficient to use this position then calculate the position before the node using
- * {@link engine.treeModel.Position.createBefore}. It is also more efficient to get the position after node by shifting
- * `previousPosition` by `length`, using {@link engine.treeModel.Position#getShiftedBy}, then calculate the position using
- * {@link engine.treeModel.Position.createAfter}.
- * @property {engine.treeModel.Position} nextPosition Next position of the iterator. For `'ELEMENT_START'` it is the first
- * position inside the element. For all other types it is the position after the item.
+ * @property {engine.treeModel.Position} previousPosition Previous position of the iterator.
+ * 		Forward iteration:
+ * 			For `'ELEMENT_END'` it is the last position inside the element. For all other types it is the position
```

Comment:
We use tab indent in docs only for code samples. Use bullet list here insted (see: https://github.com/ckeditor/ckeditor5-engine/blob/master/src/treecontroller/model-converter-builder.js#L169-L171)